README.md

```diff
@@ -91,7 +91,7 @@ catch (Exception ex)
 }
 finally
 {
-    await Log.CloseAndFlushAsync();
+    await Log.CloseAndFlushAsync(); // ensure buffering or async Sinks are flushed before app exits
```

Comment:
Open to removing this given its a lot of words...

Reply:
LGTM, maybe we could shorten it to `// ensure all logs are written before the app exits` ?

Reply:
That works, done